pages/compliance/_sub-hero.html

```diff
@@ -0,0 +1,40 @@
+<div class="row row-page-header">
+  <div class="col-xs-12">
+    <h2>Compliance in hours, not months</h2>
+    <p>
+      The <a href="https://www.cisecurity.org/benchmark/amazon_web_services/">CIS AWS Foundations Benchmark</a> is an
+      objective, of consensus-driven guideline for establishing secure infrastructure on AWS. Gruntwork's
+      production-grade, battle-tested infrastructure as code modules are built with compliance in mind. You can leverage
```

Comment:
"Compliance in mind" seems weak: e.g., it could be interpreted as, "we always keep it in mind but make no promises to actually meet the standards." Perhaps state more clearly that Gruntwork provides production-grade modules that are already pre-configured to meet the compliance benchmarks?

Reply:
Rephrased to:
> Gruntwork's production-grade, battle-tested infrastructure as code modules are built for compliance. Leverage them to achieve compliance with the Benchmark quickly and repeatably, avoiding the burden of a complex, drawn-out compliance project.